src/main.rs

```diff
@@ -149,13 +150,11 @@ fn main() -> Result<()> {
 
     // Add library dependencies if requested
     if cli.ldd {
-        match exec::get_library_dependencies(&binary_path_str) {
```

Comment:
Lets remove this from the exec module if no longer required

Reply:
Done